CHANGELOG.md

```diff
@@ -12,6 +12,27 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+## [0.25.0] - 2025-01-01
+
+### Release highlights
+
+It's the holidays, and this release was overall pretty quiet, without many major
+changes. Two select improvements:
+
+* Improvements to configuration management, including support for [conditional
+  variables](docs/config.md#conditional-variables) in config files.
+
+* Large files will no longer cause commands to fail; instead the large files
```

Comment:
I think we should say clarify saying "Large files in the working copy ..." here.

(sorry, maybe my previous comment made it less clear)